docs/operations-manual.md

```diff
@@ -88,8 +83,7 @@ helm install postgres ./metadig-postgres --namespace metadig --version=1.0.0
 
 The metadig-postgres service uses the CephFS Persistent Volume 'cephfs-metadig-pv'. Communication between the metadig-postgres pod and the PV is provided by [ceph-csi](https://github.com/DataONEorg/k8s-cluster/blob/main/storage/Ceph/Ceph-CSI.md).
 
-In addition, the PV has been made available to the Linux command line on the production control node `k8s-ctrl-1.dataone.org` via a volume mount. This volume mount
-was manually created and is available at `/mnt/k8ssubvol`. The subdirectory `postgresql` contains the metadig-postgres database files.
+In addition, the PV has been made available to the Linux command line on the production control node `k8s-ctrl-1.dataone.org` via a volume mount. This volume mount was manually created and is available at `/mnt/k8ssubvol`. The subdirectory `postgresql` contains the metadig-postgres database files.
```

Comment:
Do we still need this? I am worried that mounting this encourages out-of-band editing of the files on this disk. Why do we need it mounted here?

Reply:
I don't think we do - but I have never actually looked at that mount before. Not even sure I have direct access to the control node.